src/components/Calculator.js

```diff
@@ -15,31 +16,34 @@ const Calculator = () => {
   };
 
   return (
-    <div className="calculator">
-      <div className="display-screen">
-        {total}
-        {operation}
-        {next}
+    <div className="calc-container">
+      <Title value="title" title="Let's do math" />
```

Comment:
- [ ] @clin2on3mun great work adding the title math, however, the layout still doesn't match it would be nice if they were of the same height from the navbar section(the distance from navbar to the calculator and the distance from the navbar to the title should be equal), check the comparison below for a better understanding


Wireframe Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://github.com/clin2on3mun/MathMagician_app/assets/61408860/c1264b75-1c98-412f-9874-c9e7ba51f1e4)| ![image](https://github.com/clin2on3mun/MathMagician_app/assets/61408860/63a46125-50b3-4220-a34a-aa30ce8697cd)